docs/en/developer.md

```diff
@@ -423,6 +423,41 @@ for the advanced settings.
 
 ## Advanced features
 
+### Allowing email login on LDAP login form
+
+If you have registered `LDAPAuthenticator` (see "Configure SilverStripe Authenticators" section), you may notice the only
+accepted login is username on the form. This may be confusing if your users are more familiar logging in with using their email
+address.
+
+To allow logging in with email in addition to username, set the following configuration:
+
+```yaml
+LDAPAuthenticator:
+  allow_email_login: 'yes'
+```
+
+Note that your LDAP users logging in must have the `mail` attribute set, otherwise this will not work.
```

Comment:
1. done
2. yes, but this is an "advanced" feature and the attribute has to exist in LDAP (whatever way they get them in there, it's not really up to us)